components/display/index.rst

```diff
@@ -147,6 +148,20 @@ a ``.ttf`` file from somewhere on the Internet and create a ``font:`` section in
       # ...
 
 
+If you created a ``fonts`` subfolder, reference the font file as follows:
```

Comment:
Maybe we should just adjust the example above to have a subfolder by default.

Reply:
For the sake of keeping everything organised, I'd definitely use subfolders for fonts etc.
Maybe `_fonts` is even better than `fonts` ?

Reply:
> Maybe _fonts is even better than fonts ?

Why?

Reply:
It could work as a visual aid. Also keeps the folders up top in the explorer window.